### PR TITLE
Add db stuff to README, fix bug with PHP count() usage

### PR DESCRIPTION
**Pull request by jbraswell**

### CODE REVIEW

**Comment by EzraKuhr** [comment]
Pretty sure that denied message will fire back to the sender too after all payloads are delivered to the list

**Comment by jbraswell** [comment]
> Pretty sure that denied message will fire back to the sender too after all payloads are delivered to the list

I don't think so. The denied message is only set if the user doesn't have permissions to broadcast.

File: subscribe.php [hunk 1]
```diff
@@ -40,9 +40,8 @@
     $db->bind(":contact", $contact);
     $found = $db->single();
 
-    if (count($found) == 1) {
+    if ($found && count($found) == 1) {
         $db->query("SELECT `contact` FROM `subscribers`");
-        $db->bind(":contact", $contact);
```
Comment (jbraswell) [inline subscribe.php:45]: Binding this variable without actually using it in the query was causing an error in PHP 8.

File: subscribe.php [hunk 2]
```diff
@@ -40,9 +40,8 @@
     $db->bind(":contact", $contact);
     $found = $db->single();
 
-    if (count($found) == 1) {
+    if ($found && count($found) == 1) {
```
Comment (jbraswell) [inline subscribe.php:43]: In some PHP versions in some scenarios, `count(false)` will actually return `1`.

Comment (pjaudiomv) [reply subscribe.php:43]: That's silly 🙃

Comment (pjaudiomv) [reply subscribe.php:43]: I think I've run into this before

**Pull request merged**